### PR TITLE
pytest: prefer ‘neard’ binary name

### PR DESCRIPTION
**Pull request by mina86**

### CODE REVIEW

File: pytest/lib/branches.py [hunk 1]
```diff
@@ -68,17 +68,17 @@ def compile_current():
     branch = current_branch()
     try:
         # Accommodate rename from near to neard
-        subprocess.check_output(['cargo', 'build', '-p', 'neard'])
-
-        subprocess.check_output(['cargo', 'build', '-p', 'near-test-contracts'])
```
Comment (chefsale) [inline pytest/lib/branches.py:73]: I don't think the --bin part is needed.

File: pytest/lib/branches.py [hunk 2]
```diff
@@ -68,17 +68,17 @@ def compile_current():
...
+        subprocess.check_call(['cargo', 'build', '-p', 'neard',
+                               '--bin', 'neard'])
```
Comment (chefsale) [inline pytest/lib/branches.py:72]: I believe --bin neard is not needed, we can delete that part.

**Review by chefsale — CHANGES_REQUESTED** [review]

File: pytest/lib/branches.py (hunk 2 above)
Comment (mina86) [reply pytest/lib/branches.py]: That’s true but from my testing it does speed up the build by a couple of seconds. (We currently have `neard` and `near` binaries defined so `--bin neard` saves on linking the latter). Not a huge improvement but it’s trivially achievable so why not? Thoughts?

Comment (chefsale) [reply pytest/lib/branches.py]: I mean hard to say if that is really gonna be such an improvement, but lets keep it than. I'm fine with either.

**Review by chefsale — APPROVED** [review]

**Pull request merged**